source/renderer/util/template.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  renderTemplate: (templateString) => {
```

Comment:
That is a good addition! So if I understand you correctly, your aim here is to just pass an HTML-string which is then rendered using the template. If so … just from the top of my head, wouldn't the function name be better if it were … `renderToDOM` …? But just an idea! Anyways, it would be cool if you could copy any one of the headers from any of the files and adapt it to this utility function so that it's clear who did it and some other information necessary for the codebase!.

Reply:
I'd say that it renders a Javascript template literal into DOM tree. I find it more important what to render rather than what to render into. If that make sense. However, I'm not attached to the current name, so I can rename it. So if you think, the return type is more important, I'd rename this function to `createNodeTree` or `createElementTree`.

About the file headers, I more see them as a liability, because the git log is more accurate source for [who change what](https://www.git-scm.com/docs/git-blame) and I usually move functions around as needed. This can leave files with stale headers.